airflow/providers/amazon/aws/operators/s3_delete_objects.py

```diff
@@ -45,6 +45,8 @@ class S3DeleteObjectsOperator(BaseOperator):
     :type prefix: str
     :param aws_conn_id: Connection id of the S3 connection to use
     :type aws_conn_id: str
+        (Adding `aws_conn_id` param to template_fields so that it can be overriden using jinja template from Dags
+         This feature can be useful if user wants to update/override the aws_conn_id for some kind of Dag Isolation etc)
```

Comment:
`aws_conn_id` wasn't added below

Reply:
yes you are right , i forgotten in first commit then later realized and added as part of the second commit..